### ISSUE TITLE
Long-running processes (>5s) may be killed by github hooks, and therefore should be avoided

### ISSUE THREAD

**Issue by Ocramius** [Roave/SecurityAdvisoriesBuilder#8]
As discussed with @fabpot in private, the hook `https://roave-security-advisories.herokuapp.com/` works, but the github hooks report a 404 error.

From my logs I see something like:

```
sock=client at=warning code=H27 desc="Client Request Interrupted" method=POST path="/" host=roave-security-advisories.herokuapp.com request_id=abc fwd="192.30.253.29" dyno=web.1 connect=1ms service=7489ms status=499 bytes= protocol=https 
```

So the client is indeed aborting (status code 499 on our side).

It is possible to fix this by making the endpoint "silent" and always producing an "OK" to the client, then terminating the HTTP socket and continuing execution in background.

**Comment by Ocramius** [comment]
This is now solvable by using github actions instead of a callback

**Comment by Ocramius** [comment]
Proper solution here: run GitHub actions (once an hour) instead of relying on a webapp deployment.

**Comment by slash3b** [comment]
Can this one be closed since we already have githuib actions hourly cron that does build and push to SecurityAdvisories  #128 ?

**Comment by Ocramius** [comment]
This is indeed solved in #128!
